TitanRobotics2024/src/main/java/frc/robot/Subsystem/DriverController.java

```diff
@@ -14,21 +20,43 @@ public static DriverController getInstance()
 
     public DriverController()
     {
-        
+        this.xboxController = new XboxController(PortMap.XBOX_DRIVER_CONTROLLER.portNumber);       
     }
 
-    @Override
-    public void start() 
-    {
-        // TODO Auto-generated method stub
-        throw new UnsupportedOperationException("Unimplemented method 'start'");
+    public double getStick(ButtonMap stickAxis) 
+    {  if (this.xboxController != null)
+        {   try {
+            switch(stickAxis)
+            {
+                case XboxLEFTSTICKX:
+                    return xboxController.getRawAxis(0);
+                case XboxLEFTSTICKY:
+                    return xboxController.getRawAxis(1);
+                case XboxRIGHTSTICKX:
+                    return xboxController.getRawAxis(4); 
+                case XboxRIGHTSTICKY:
+                    return xboxController.getRawAxis(5); 
+                default:
+                    return 0;
+            }
+        }  
+        catch (Exception AxisNotFound) 
+        {
+            SmartDashboard.putString("ControllerError", "AxisNotFound");
+            return 0;
+        }
+        } 
+        else 
+        {
+        return 0;
+        }     
     }
 
     @Override
     public void update() 
     {
         // TODO Auto-generated method stub
-        throw new UnsupportedOperationException("Unimplemented method 'update'");
+        // throw new UnsupportedOperationException("Unimplemented method 'update'");
```

Comment:
Remove commented out code

Reply:
If they think they might use it later what should they do?